.buildkite/pipeline.yml

```
@@ -61,16 +61,7 @@ steps:
     if: build.branch == "trunk" || build.tag != null
 
   - label: "Publish to S3"
```

Nitpick: I wonder if it'd help clarity (i.e. avoid confusing this with the publication of prototype builds) to rename this step to:
```suggestion
  - label: "Publish SDK to S3"
```